sentences/de/_common.yaml

```
@@ -69,9 +69,9 @@ lists:
         out: "lock"
   lock_states:
     values:
-      - in: "(([ab|zu]ge|ver)schlossen|verriegelt)"
+      - in: "(([ab|zu]ge|ver)schlossen|verriegelt|((ab|zu)ge|ver)sperrt)"
```

```suggestion
      - in: "(([ab|zu]ge|ver)(schlossen|sperrt)|verriegelt)"
```
this allows:
- abgeschlossen
- zugeschlossen
- geschlossen
- verschlossen
- abgesperrt
- zugesperrt
- gesperrt
- versperrt
- verriegelt